website/src/pages/index.js

```diff
@@ -55,15 +55,15 @@ function Home() {
               <li>
                 {' '}
                 <Link href="/jb2/blog" variant="contained">
-                  Download latest release
+                  Download latest web release
                 </Link>
               </li>
               <li>
                 <Link
                   href={siteConfig.customFields.currentLink}
                   variant="contained"
                 >
-                  Browse demo instance
+                  Browse web demo
```

Comment:
"web demo" doesn't sound that good to me. I like demo instance

Reply:
how about 'web demo instance'

Reply:
I guess so. 

It might also be nice to link out to our sample data such as "Browse demo instance (human instance with basic annotation, human instance with sample data tracks)" to link to ?config=test_data/config_demo.json  too

Reply:
It is not popular but I'd also just link the volvox sample data too if it were me